packages/web-runtime/src/components/MessageBar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <oc-notifications :position="notificationPosition">
+  <oc-notifications position="default">
```

Comment:
Instead of having the `position` prop here at all you could change the default value of the prop to `default` in the `OcNotifications.vue` component. Kind of ridiculous that `OcNotifications` has `top-center` as default while `default` is a valid value. 😆

Reply:
hahah, yess I was baffled :D